homeassistant/components/lupusec/__init__.py

```diff
@@ -85,10 +127,16 @@ def __init__(self, username, password, ip_address, name):
 class LupusecDevice(Entity):
     """Representation of a Lupusec device."""
 
-    def __init__(self, data, device):
+    _attr_has_entity_name = True
```

Comment:
I think it's best to leave the has entity name and device info changes for a later PR so we can focus on the config flow in this PR

Reply:
Removed.